frontend/src/app/selectors/experiment.js

```diff
@@ -0,0 +1,30 @@
+import { createSelector } from 'reselect';
+import moment from 'moment';
+
+
+// Return all experiments from the store, sorted.
+export const allExperimentSelector = createSelector(
+  store => store.experiments.data,
+  experiments => (
+    experiments.sort((a, b) => {
```

Comment:
FWIW, `[].sort()` [modifies the array in place](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort) and returns the original array. That's why I did [the hack with `concat`](https://github.com/mozilla/testpilot/pull/1466/files#diff-4178a3feb17552468ad69a80a4b66b2cL9) in the old code to copy the array to `experimentsSorted` before sorting.